PianoFromAbove/GameState.cpp

```diff
@@ -660,19 +666,12 @@ void MainScreen::InitState()
         int width = rect.right - rect.left;
         int height = rect.bottom - rect.top;
 
+        //Running ffmpeg
         char buf[1024] = {};
-        snprintf(buf, sizeof(buf), "Waiting for connection... (%d x %d)", width, height);
-        SetWindowTextA(g_hWnd, buf);
-        m_hVideoPipe = CreateNamedPipe(TEXT("\\\\.\\pipe\\pfadump"),
-            PIPE_ACCESS_OUTBOUND,
-            PIPE_TYPE_BYTE | PIPE_WAIT,
-            PIPE_UNLIMITED_INSTANCES,
-            static_cast<DWORD>(width * height * 4 * 120),
-            0,
-            0,
-            nullptr);
+        snprintf(buf, sizeof(buf), "%s&cd \"%s\"&md \"%s\\PFA_Framedump\"&start cmd /k \"ffmpeg -r 60 -f rawvideo -s %dx%d -pixel_format rgb32 -i \"async:\\\\.\\pipe\\pfadump\" -c:v h264_nvenc -qp 0 ^\"%s\\PFA_Framedump\\Output.mkv^\"\"", GetExePath().substr(0, 2).c_str(), GetExePath().c_str(), GetExePath().c_str(), width, height, GetExePath().c_str());
```

Comment:
This kind of thing really has to be configurable. Like, what if someone wasn't on a NVIDIA GPU?